### DISCUSSION TITLE
Netflix股價暴跌7成，面對頹勢Netflix將如何挽回訂戶？

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#448] [General]
Netflix股價暴跌7成，面對頹勢Netflix將如何挽回訂戶？
越來越少人看了嗎，我自己是比較常看Disney+拉
https://tw.news.yahoo.com/netflix%E8%82%A1%E5%83%B9%E6%9A%B4%E8%B7%8C7%E6%88%90-%E9%9D%A2%E5%B0%8D%E9%A0%B9%E5%8B%A2netflix%E5%B0%87%E5%A6%82%E4%BD%95%E6%8C%BD%E5%9B%9E%E8%A8%82%E6%88%B6-124525581.html

**Comment by brian431** [comment]
老實說Disney+跟Netflix的影片池大小是完全不能比的。使用者介面更是差到不行。唯一的優勢是擁有像漫威、星際大戰等版權。

**Comment by boson13579** [comment]
對於加入廣告，來降低使用費率我個人是持支持態度的，就像巴哈動畫瘋一樣，我想一定很多人也跟我一樣是沒有購買會員，每次都看廣告的吧！

**Comment by lx3211** [comment]
同意樓上，我也很願意接受廣告，每次巴哈都放著廣告去洗手間xd

**Comment by Chenszuhong** [comment]
我自己覺得因為帳號密碼可以借用所以少了很多用戶

**Comment by limitx0** [comment]
很同意加入廣告
有時候我只想看幾部戲，就會覺得訂閱沒那麼划算
